solutions/standard/main.tf

```diff
@@ -232,3 +232,81 @@ data "ibm_database_connection" "existing_connection" {
   user_id       = data.ibm_database.existing_db_instance[0].adminuser
   user_type     = "database"
 }
+
+########################################################################################################################
+# Code Engine Kibana Dashboard instance
+########################################################################################################################
+
+data "http" "es_metadata" {
```

Comment:
We should double check those 2 scenarios:
1. Elastic search creation - the ES deployment typically takes a while to be up and running, there is a risk that this data lookup will fails while done against ES
2. ES configured for private network only. The code should in theory works when running from Schematics (access to private backbone), but I think this should be explicitly checked.

Reply:
I would also give a way for the user to bypass this call by allowing the user to input the exact version of their ES database through an option input variable.

Reply:
- I ran multiple times and noticed the data block is runnning only after ES enstance is deployed
<img width="999" alt="image" src="https://github.com/user-attachments/assets/9b7f9532-15fc-4de2-9598-8d6141d854a7">
- Also added variable for the user to pass es full version.